api/team.go

```diff
@@ -801,6 +830,10 @@ func updateTeam(c *Context, w http.ResponseWriter, r *http.Request) {
 
 	oldTeam.Sanitize()
 
+	message := model.NewWebSocketEvent(model.WEBSOCKET_EVENT_UPDATE_TEAM, "", "", "", nil)
```

Comment:
This is to be able to update the UI when a team goes from close to open and vice-versa